<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>395</width>
    <height>340</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Hello Minecraft! Launcher 0.2 by huanghongxun</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QPushButton" name="btnRun">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>280</y>
      <width>151</width>
      <height>27</height>
     </rect>
    </property>
    <property name="text">
     <string>开始Minecraft之旅！</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="txtMinecraftPath">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>40</y>
      <width>381</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>101</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Minecraft路径</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnSelMCPath">
    <property name="geometry">
     <rect>
      <x>318</x>
      <y>10</y>
      <width>71</width>
      <height>27</height>
     </rect>
    </property>
    <property name="text">
     <string>手动选择</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="txtJavaPath">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>100</y>
      <width>381</width>
      <height>27</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="btnSelJavaPath">
    <property name="geometry">
     <rect>
      <x>318</x>
      <y>70</y>
      <width>71</width>
      <height>27</height>
     </rect>
    </property>
    <property name="text">
     <string>手动选择</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>75</y>
      <width>101</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Java路径</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>135</y>
      <width>101</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>最大内存</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="txtMaxMemory">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>160</y>
      <width>381</width>
      <height>27</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="btnSelGoodMem">
    <property name="geometry">
     <rect>
      <x>318</x>
      <y>130</y>
      <width>71</width>
      <height>27</height>
     </rect>
    </property>
    <property name="text">
     <string>最佳内存</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>195</y>
      <width>101</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>游戏名称</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="txtPlayerName">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>220</y>
      <width>381</width>
      <height>27</height>
     </rect>
    </property>
   </widget>
   <widget class="QComboBox" name="cboVersion">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>280</y>
      <width>161</width>
      <height>27</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>285</y>
      <width>67</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>游戏版本</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_6">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>250</y>
      <width>101</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>启动模式</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnSelHome">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>10</y>
      <width>71</width>
      <height>27</height>
     </rect>
    </property>
    <property name="text">
     <string>默认路径</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="rdo32Bit">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>250</y>
      <width>61</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string>32位</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="rdo64Bit">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>250</y>
      <width>61</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string>64位</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="rdoNormal">
    <property name="geometry">
     <rect>
      <x>320</x>
      <y>250</y>
      <width>71</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string>不强制</string>
    </property>
    <property name="checked">
     <bool>true</bool>
    </property>
    <property name="autoRepeat">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="btnMakeSh">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>310</y>
      <width>151</width>
      <height>27</height>
     </rect>
    </property>
    <property name="text">
     <string>制作一键启动</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnInclude">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>310</y>
      <width>99</width>
      <height>27</height>
     </rect>
    </property>
    <property name="text">
     <string>导入旧版本</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnManageVersion">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>310</y>
      <width>99</width>
      <height>27</height>
     </rect>
    </property>
    <property name="text">
     <string>修改版本信息</string>
    </property>
   </widget>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
